src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+TraceWarningPackageDiscovery.swift

```diff
@@ -0,0 +1,131 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+@testable import ENA
+import Foundation
+import XCTest
+
+final class HTTPClientTraceWarningPackageDiscovery: XCTestCase {
```

Comment:
```suggestion
final class HTTPClientTraceWarningPackageDiscoveryTests: XCTestCase {
```

Don't forget to rename the File, otherwise Sonar will be confused :D

Reply:
 😅